test/oauth2_session_test.py

```diff
@@ -45,3 +45,10 @@ def test_fetch_token(self, mocked_func):
             include_client_id=True,
             code='code',
         )
+
+    def test_user_agent_header(self):
+        expected_ua_header = {
+            'User-Agent': f'bynder-python-sdk/{SDK_VERSION}'
+        }
+        # The UA header is contained within the session headers
+        assert expected_ua_header.items() <= self.session.headers.items()
```

Comment:
```suggestion
        assert UA_HEADER.items() <= self.session.headers.items()
```
I'd say no no need to recreate the header here since it's pretty much a constant.